sapp/tests/warning_messages_test.py

```diff
@@ -0,0 +1,98 @@
+import json
```

Comment:
Our linter is bugging us to include the license

```suggestion
# Copyright (c) Facebook, Inc. and its affiliates.
#
# This source code is licensed under the MIT license found in the
# LICENSE file in the root directory of this source tree.

import json
```

Reply:
added